spinnman/transceiver.py

```diff
@@ -603,7 +603,7 @@ def _make_chip(self, chip_details):
             ip_address=chip_details.ip_address)
         return chip
 
-    def _update_machine(self):
+    def _update_machine(self, adjust_so_fpgas_are_virtual_chips):
```

Comment:
Want to rename this, or do it a different way - may remove this for now as a parameter...